FileWritingObjects/FileWritingObjects/NumSet.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace FileWritingObjects
+{
+    public class NumSet
+    {
+        public LinkedList<int> Set;
+
+        public NumSet(params int[] variable)
+        {
+            Set = new();
+            foreach(int i in variable)
+            {
+                if (!Set.Contains(i))
+                    Set.AddFirst(i);
+            }
+        }
+
+        public override string ToString()
+        {
+            string res = "";
```

Comment:
```suggestion
            string result = "";
```
Do not use abbreviated variable names.